include/Framework/EventHeader.h

```diff
@@ -95,6 +111,22 @@ class EventHeader {
    */
   double getWeight() const { return weight_; }
 
+  /**
+   * Set the number of tries it took to generate this event
+   * @note This is used within Framework during Production Mode
+   * and so changing it in a downstream Producer will be confusing
+   * (even if it runs properly).
+   *
+   * @param[in] t the number of tries
+   */
+  void setTries(int t) { tries_ = t; }
```

Comment:
```suggestion
  void setTries(int tries) { tries_ = tries; }
```
avoid single-character variable names to avoid conflicts.